articles/storage/blobs/storage-dotnet-how-to-use-blobs.md

```diff
@@ -20,385 +20,370 @@ ms.author: tamram
 
 [!INCLUDE [storage-selector-blob-include](../../../includes/storage-selector-blob-include.md)]
 
-[!INCLUDE [storage-check-out-samples-dotnet](../../../includes/storage-check-out-samples-dotnet.md)]
+This tutorial will show you how to use [Azure Blob Storage](https://docs.microsoft.com/azure/storage/blobs/storage-blobs-introduction) with ASP.NET and Visual Studio 2017. It will show how to upload, list, download, and delete blobs in the context of a simple Web API project that works with images.
```

Comment:
So one thing that I believe trips people up is that they don't know it is compatible with other project types. I would maybe point out right here that this same exact code and libraries are compatible with other .NET project types such as ASP.NET Core, Xamarin, Windows Desktop Apps, .NET Core, and More.

Reply:
I'll make mention that the APIs covered here are available for other platforms.